install_base.sh

```diff
@@ -314,6 +314,8 @@ for arg in "$@"; do
         # gettext for autopoint
         pacman_packages+=(gettext autoconf libtool bison)
 
+        git clone https://github.com/alpinelinux/alpine-chroot-install.git --depth=1 -C ${LISA_HOME}/tools
```

Comment:
Looks like we miss the relevant qemu packages for userspace emulation, they should be added to apt_packages and pacman_packages (the latter is for archlinux)

Reply:
The alpine script should install `qemu-user-static` automatically, did you hit an error ?

Reply:
I did not notice the script was doing it. Would still be a better idea to add it to install_base.sh since:
1. this lists our actual dependencies (important for review and if we want to enable LISA on other distro)
2. the installation procedure is done so that after install_base.sh, all things that are pulled have an impact that is strictly local to the repository, with absolutely no global change. This is important in order to avoid unexpected pollution of users' setup.